.github/workflows/ci.yml

```diff
@@ -2,34 +2,37 @@ name: CI
 
 on:
   push:
-  pull_request_target:
+  pull_request:
   schedule:
     - cron: "6 20 * * 6"
 
 jobs:
   test:
     runs-on: ubuntu-latest
-    container: ${{matrix.image}}
     strategy:
       fail-fast: false
       matrix:
-        image:
-          - ruby:2.5.0
-          - ruby:2.5
-          - ruby:2.6.0
-          - ruby:2.6
-          - ruby:2.7.0
-          - ruby:2.7
-          - jruby:9.2
-        include:
-          - image: ruby:rc
-            continue-on-error: true
+        ruby:
+          - 2.5
```

Comment:
My understanding is this means "the latest in 2.5 line". We specifically want this and 2.5.0. Is that possible?

Reply:
Yes it is:

```
- 2.5.0
- 2.5
```